pkgs/applications/misc/azuredatastudio/default.nix

```diff
@@ -0,0 +1,118 @@
+{ stdenv
+, lib
+, fetchurl
+, makeWrapper
+, libuuid
+, libunwind
+, icu
+, openssl
+, zlib
+, curl
+, at-spi2-core
+, at-spi2-atk
+, gnutar
+, atomEnv
+, kerberos
+}:
+
+# from justinwoo/azuredatastudio-nix 
+# https://github.com/justinwoo/azuredatastudio-nix/blob/537c48aa3981cd1a82d5d6e508ab7e7393b3d7c8/default.nix
+
+let
+  dynamic-linker = stdenv.cc.bintools.dynamicLinker;
```

Comment:
This is only used one out of two times. It should either be consistently used or remove, I think removing it would be better.